lib/UserBackend.php

```diff
@@ -474,6 +479,10 @@ public function updateAttributes($uid,
 				);
 				$this->setDisplayName($uid, $newDisplayname);
 			}
+
+			if ($newQuota !== null) {
+				$user->setQuota($newQuota);
```

Comment:
should we do some additional sanity checks here, e.g. if it is an integer and >0? Or does setQuota() checks it already?

Reply:
Set quota does all kind of magic parsing. Like if it is an int it is in bytes. But you can also pass 100MB etc.

As far as I can tell we also just pass it on in the LDAP code.